features/product/managing_products/product_validation.feature

```diff
@@ -5,14 +5,14 @@ Feature: Products validation
     I want to be prevented from adding it without specifying required fields
 
     Background:
-        Given the store is available in "English (United States)"
+        Given the store operates on a single channel in "United States"
         And I am logged in as an administrator
 
     @ui
     Scenario: Adding a new simple product without specifying its code
-        Given I want to create a new simple product
-        When I name it "Dice Brewing" in "English (United States)"
-        And I set its price to "$10.00"
+        When I want to create a new simple product
+        And I name it "Dice Brewing" in "English (United States)"
```

Comment:
Currently we use `Given I want to...` and `When I name it ...` everywhere I think.